### PR TITLE
Updates the top level navbar and reorganizes existing content to fit

### PR DESCRIPTION
**Pull request by emanic**
## Description

- Updates the topic titles in the top-level navbar
- Reorganizes the existing content to fit under these new headings/categories
- Adds attributions

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
None required
```

### CODE REVIEW

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 6895bd49fb5b10418645403d771ca674a4e54f0f

https://deploy-preview-2372--calico.netlify.com

**Comment by caseydavenport** [comment]
Deploy preview for *calico* ready!

Built with commit 6895bd49fb5b10418645403d771ca674a4e54f0f

https://deploy-preview-2372--calico.netlify.com

File: _data/master/navbars/networking.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+bigheader: "Networking"
+path: /networking
+order: 2
+toc:
+- title: Networking
```
Comment (caseydavenport) [inline _data/master/navbars/networking.yml]: I think this section is missing the new stuff from v3.5 re: IPAM based on topology.

File: master/getting-started/kubernetes/installation/config-options.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Customizing the manifests
+title: Customizing the manifests (advanced)
```
Comment (caseydavenport) [inline master/getting-started/kubernetes/installation/config-options.md]: Don't know if we want to characterize this as "advanced" - it's probably a normal thing to do.

File: master/maintenance/openstack-upgrade.md [hunk 3]
```diff
@@ -75,14 +75,10 @@ This part of the upgrade procedure varies slightly according to which operating
    calico-upgrade complete
    ```
 
-1. Remove any existing `calicoctl` instances and [install the new `calicoctl`](/{{page.version}}/usage/calicoctl/install).
+1. Remove any existing `calicoctl` instances and [install the new `calicoctl`](../getting-started/calicoctl/install).
```
Comment (caseydavenport) [inline master/maintenance/openstack-upgrade.md]: This doc seems to have explicit references to v3.1 in it. 

For example, https://github.com/projectcalico/calico/pull/2372/files#diff-b7e3a822caacd3e36fffdff4b3c05921R100

File: master/maintenance/kubernetes-upgrade.md [hunk 4]
```diff
@@ -157,8 +149,4 @@ and your datastore type.
    upgrade by running `calico-upgrade complete`. After this, you can once again schedule
```
Comment (caseydavenport) [inline master/maintenance/kubernetes-upgrade.md]: I don't think this block is relevant any more since we've removed the v2.6 -> v3 upgrade content.

Comment (caseydavenport) [reply master/maintenance/kubernetes-upgrade.md]: Also wonder if we should explicitly call out at the top of this page that it covers upgrade between Calico v3.x versions only.

File: master/maintenance/openstack-upgrade.md (hunk 3 above)
Comment (emanic) [reply master/maintenance/openstack-upgrade.md]: Yes, while fixing the broken links, I ended up correcting some errors in older directories.

File: master/reference/etcd-rbac/kubernetes.md [hunk 5]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Using etcd RBAC to segment Kubernetes and Calico
+title: Segmenting etcd on Kubernetes (basic)
```
[the remark this answers is not in the document]
Comment (emanic) [reply master/reference/etcd-rbac/kubernetes.md]: I personally agree with you one hundred percent, if you wanted to take this up with AP perhaps we could do that.

File: _data/master/navbars/networking.yml (hunk 1 above)
Comment (emanic) [reply _data/master/navbars/networking.yml]: Yes, I had been working to update this PR around that so it's still a little in progress. Now we have v3.5, I'll need to rebase again, as you noted.

File: master/maintenance/openstack-upgrade.md (hunk 3 above)
Comment (caseydavenport) [reply master/maintenance/openstack-upgrade.md]: This isn't an older directory though, this is master, so I don't think it should be saying stuff like "Use the following command on the compute nodes to confirm that Felix has upgraded to v3.1.x."

Comment (emanic) [reply master/maintenance/openstack-upgrade.md]: That's a good point, let me look into this.

File: _layouts/docwithnav.html [hunk 6]
```diff
@@ -99,15 +99,38 @@
           </div>
           <div id="tigera-navbar" class="navbar-collapse collapse">
             <ul class="nav navbar-nav navbar-right">
+
```
Comment (caseydavenport) [inline _layouts/docwithnav.html]: nit: I think there's a corresponding line in RELEASING.md that needs to be updated for this change

> Add TWO new <li> entries to the <span class="dropdown"> in _layouts/docwithnav.html file.

TWO probably goes to ONE now?

**Pull request merged**